### PR TITLE
Fix some issues of elasticsearch junit rule

### PR DESCRIPTION
**Pull request by Borjianamin98**
+ Fixed an issue where temporary files were not deleted
  + Properly adjust the replica and shards to to prevent the cluster from remaining yellow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,14 @@
 # Elasticsearch Rule
+[![Tests](https://github.com/sahabpardaz/elasticsearch-rule/actions/workflows/maven.yml/badge.svg?branch=es-7)](https://github.com/sahabpardaz/elasticsearch-rule/actions/workflows/maven.yml)
```
Comment (dataeng1n33r) [inline README.md]: branch=master

File: pom.xml [hunk 2]
```diff
@@ -6,19 +6,28 @@
...
+            <id>jitpack.io</id>
+            <url>https://jitpack.io</url>
+        </repository>
+    </repositories>
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <elasticsearch.version>6.2.4</elasticsearch.version>
+        <cleanup.version>1.0.1</cleanup.version>
     </properties>
```
Comment (dataeng1n33r) [inline pom.xml:26]: Add sonarcloud properties

File: src/main/java/ir/sahab/elasticsearchrule/ElasticsearchRule.java [hunk 3]
```diff
@@ -56,28 +64,53 @@ protected void before() throws IOException, NodeValidationException, ExecutionEx
         // ReindexPlugin is necessary for making "delete by query" available.
         server = new TestNode(settings, Arrays.asList(Netty4Plugin.class, ReindexPlugin.class));
         server.start();
-        server.client().admin().cluster().prepareHealth().setWaitForYellowStatus().execute().get();
+        server.client().admin().cluster().prepareHealth().setWaitForGreenStatus().execute().get();
```
Comment (dataeng1n33r) [inline src/main/java/ir/sahab/elasticsearchrule/ElasticsearchRule.java]: Also check the response.status()
If this command gets time out, no error will be raised.
Another approach is to remove execute() like setWaitForGreenStatus().get(); Maybe in this case an error is thrown for time out.

File: src/test/java/ir/sahab/elasticsearchrule/ElasticsearchRuleTest.java [hunk 4]
```diff
@@ -66,13 +72,14 @@ public void testAddress() {
...
-            throw new AssertionError("Cannot get the elasticsearch server address " + elasticsearchHost + ".", e);
+            throw new AssertionError("Cannot get the elasticsearch server address: " + elasticsearchHost, e);
         }
         Settings settings = Settings.builder().put("cluster.name", ELASTICSEARCH_CLUSTER_NAME).build();
         TransportClient internalTransportClient = new PreBuiltTransportClient(settings);
         internalTransportClient.addTransportAddress(new TransportAddress(elasticsearchInetAddress, elasticsearchPort));
         String indexName = "twitter2";
-        CreateIndexResponse createIndexResponse = internalTransportClient.admin().indices().prepareCreate(indexName).get();
+        CreateIndexResponse createIndexResponse = internalTransportClient.admin().indices()
+                .prepareCreate(indexName).get();
```
Comment (dataeng1n33r) [inline src/test/java/ir/sahab/elasticsearchrule/ElasticsearchRuleTest.java]: Why did you put it in a new line?

**Review by dataeng1n33r — CHANGES_REQUESTED** [review]

File: src/test/java/ir/sahab/elasticsearchrule/ElasticsearchRuleTest.java (hunk 4 above)
Comment (Borjianamin98) [reply src/test/java/ir/sahab/elasticsearchrule/ElasticsearchRuleTest.java]: Line length was more than 120.

File: src/main/java/ir/sahab/elasticsearchrule/ElasticsearchRule.java (hunk 3 above)
Comment (Borjianamin98) [reply src/main/java/ir/sahab/elasticsearchrule/ElasticsearchRule.java]: As we talk.

**Pull request merged**